packages/bot-web-ui/src/components/blockly-loading/blockly-loading.tsx

```diff
@@ -13,10 +13,10 @@ const BlocklyLoading = ({ is_loading }) => (
     </>
 );
 
-BlocklyLoading.propTypes = {
-    is_loading: PropTypes.bool,
+type TBlocklyLoadingProps = {
```

Comment:
because of typescript convention to keep types/intefaces on top of component i think we can move this on top of component definition.